File: back-end/src/main/kotlin/com/sorsix/skopjecarrent/model/Car.kt
package com.sorsix.skopjecarrent.model

import com.sorsix.skopjecarrent.model.enum.Seats
import com.sorsix.skopjecarrent.model.enum.Transmission
import javax.persistence.*

@Entity
data class Car(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long,
    val model: String,
    val priceForADay: Int,
    val image: String,
    val withInsurance: Boolean,
    val withAirConditioner: Boolean,
    @Enumerated(value = EnumType.STRING)
    val seats: Seats,
    @Enumerated(value = EnumType.STRING)
    val transmission: Transmission,
    val sumRating: Int,
    val numReviews: Int
)

File: front-end/src/app/model/Car.ts
import { NumberInput } from "@angular/cdk/coercion";
import { Seats } from "./enum/Seats";
import { Transmission } from "./enum/Transmission";

export interface Car {
    id: number,
    model: string,
    priceForADay: number,
    image: string,
    withInsurance: Boolean,
    withAirConditioner: Boolean,
    seats: Seats,
    transmission: Transmission,
    totalPrice: number
    sumRating: number
    numReviews: number
    totalRating: string
}